asctb-api/src/functions/api.functions.ts

```diff
@@ -2,6 +2,24 @@
 import { arrayNameMap, createObject, HEADER_FIRST_COLUMN, objectFieldMap, Row } from '../models/api.model';
 import { fixOntologyId } from './lookup.functions';
 
+/**
+ *
+ * @param url  of the sheet to compare
+ * @returns a csvUrl
```

Comment:
Remove or update this doc string (also there should not be a blank line between the doc string and the function (ie line 10 below). Otherwise, looks fine.